components/PostModal/PostModal.js

```diff
@@ -5,22 +5,20 @@ import { useForm, Controller } from 'react-hook-form';
 import InputField from '../InputField/InputField';
 import { Form } from './styled';
 
-const PostModal = ({ isModalOpen, closeModalHandler, editedPost, handleEditPostAsync, handleAddPost }) => {
+const PostModal = ({ title, isModalOpen, closeModalHandler, editedPost, handleAddPost, handleEditPostAsync, button }) => {
```

Comment:
Naming issue: `button` name is confusing. Should be `submitButtonText` or `submitButtonTitle`